code/obj/item/gun/ammo.dm

```diff
@@ -475,13 +492,15 @@
 
 	smg
 		name = "9mm SMG magazine"
+		desc = "An extended 9mm magazine."
 		amount_left = 30.0
 		max_amount = 30.0
 		ammo_type = new/datum/projectile/bullet/bullet_9mm/smg
 
 /obj/item/ammo/bullets/nine_mm_NATO
-	sname = "9mm NATO"
-	name = "9mm magazine"
+	sname = "9mm polymer" 
+	name = "9mm polymer magazine"
```

Comment:
@virvatuli-gh this good?

Reply:
```suggestion
	sname = "9mm frangible" 
	name = "9mm frangible magazine"
```
Would be a more fun technical description while still being accurate to the description and purpose of the rounds. Could also consider making the description allude to them being issued so the NTSO doesn't blow holes in the station with hardnose rounds. It's all good either way though, both work.